ts/user-picker/user-picker.service.ts

```diff
@@ -75,9 +75,7 @@ module lui.userpicker {
 
 		public getMe(): ng.IPromise<IUserLookup> {
 			if (this.meCache !== undefined) {
-				let dfd = this.$q.defer();
-				dfd.resolve(this.meCache);
-				return dfd.promise;
+				return this.$q.resolve(this.meCache);
```

Comment:
I didn't know you could do this, nice! But maybe you should use the generic version?

``` Typescript
this.$q.resolve<IUserLookup>(this.meCache);
```